packages/bruno-app/src/components/ResponsePane/index.js

```diff
@@ -34,6 +44,63 @@ const ResponsePane = ({ rightPaneWidth, item, collection }) => {
 
   const response = item.response || {};
 
```

Comment:
I think having this code makes sense to be isolated into two components
`ResponseError` and `ResponseErrorIcon`